fsleyes-plugin-shimming-toolbox/fsleyes_plugin_shimming_toolbox/tabs/b0shim_tab.py

```diff
@@ -797,3 +801,54 @@ def create_sizer_max_intensity(self, metadata=None):
         )
         sizer = self.run_component_mi.sizer
         return sizer
+
+    def create_sizer_coil_constraints(self, metadata=None):
+        path_output = os.path.join(__CURR_DIR__, "output_coil_config")
+        input_metadata = [
+            {
+                "button_label": "Name",
+                "required": True,
+                "name": "name",
+            },
+            {
+                "button_label": "Number of channels",
+                "required": True,
+                "name": "channels",
+            },
+            {
+                "button_label": "Minimum current",
+                "required": True,
+                "name": "min",
+            },
+            {
+                "button_label": "Maximum current",
+                "required": True,
+                "name": "max",
+            },
+            {
+                "button_label": "Maximum sum of current",
+                "required": True,
+                "name": "max-sum",
+            },
+            {
+                "button_label": "Units",
+                "required": False,
+                "name": "units",
+            },
+            {
+                "button_label": "Output File",
+                "button_function": "select_folder",
+                "default_text": os.path.join(path_output, "config_file.nii.gz"),
+                "name": "output",
+                "required": True
+            }
+        ]
+        component_input = InputComponent(self, input_metadata)
+
+        run_component = RunComponent(
+            panel=self,
+            list_components=[component_input],
+            st_function="st_create_coil_profiles constraint-file",
```

Comment:
```suggestion
            st_function="st_create_coil_profiles config-file",
```
If accepted, need to change function name too

Reply:
I feel like we should use config because that is what we ask the user when using the shimming functions

Reply:
It becomes confusing since `st_create_coil_profiles config-file` is used as the config file for b0 shimming rather than the config file of the `st_create_coil_profiles from-fieldmaps`... So I'll opt for renaming the config file for B0 shimming to constraint files.